types/sequelize/v3/index.d.ts

```diff
@@ -10,12 +10,11 @@
 
 // Based on original work by: samuelneff <https://github.com/samuelneff/sequelize-auto-ts/blob/master/lib/sequelize.d.ts>
 
-/// <reference types="validator" />
-
-
 import * as _ from "lodash";
 import * as Promise from "bluebird";
 
+type ValidatorJS = typeof import('validator')
```

Comment:
Prefer to keep the same import way as other [imports](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40192/files#diff-b6b16ead6d43beabdd6bd9ef300fbd17R14)
```suggestion
import * as ValidatorJS from "validator";
```